packages/utils/src/assetData/baseAssets.ts

```diff
@@ -121,6 +121,21 @@ export const thorchain: Readonly<Asset> = Object.freeze({
   relatedAssetKey: null,
 })
 
+export const tcy: Readonly<Asset> = Object.freeze({
+  assetId: caip.thorchainAssetId,
+  chainId: caip.thorchainChainId,
+  name: 'TCY',
+  networkName: 'THORChain',
+  symbol: 'TCY',
+  precision: 8,
+  color: '#33FF99',
+  icon: 'https://raw.githubusercontent.com/shapeshift/web/feat_tcy_claims/scripts/generateAssetData/thorchain/icons/tcy-icon.png',
```

Comment:
Note: We'll want to make this `main` once this gets merged

Reply:
I think since this is going to make it to develop, we're good to already make this `develop`!

Reply:
4802bfca9

Reply:
🧠